gateway/api/v1/allowlist.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Curious about why do you opt by a JSON file to manage this, Paul? The problem that I see with this approach is that every time we want to modify this `white list` we are going to need to release a new version, aren't we?

What steps do we need to do in case we want to update this list in production, for example?

Reply:
Right now, just to make sure the whole process works (and also to be able to test).

Ultimately the allowlist should be mounted as a configmap on the gateway pod (meaning that file would live in the gateway helm templates directory), and since we deploy helm from the prod branch, all we'd need to do is update the prod branch to load the new configmap to the cluster.

Reply:
I see, so the idea is that we build a configmap and we get it using the environment variable, right?

Reply:
Pretty much. 
Though we wouldn't access as an envvar, but just [mount the configmap as a file on the pod](https://kubernetes.io/docs/concepts/configuration/configmap/#using-configmaps-as-files-from-a-pod). And we can just make sure the mount point is the same as the [default value](https://github.com/Qiskit/qiskit-serverless/blob/f490ec2abf94048876b984cf01d9aa3f3bd10ed9/gateway/main/settings.py#L357), and then we're good to go.